query-engine/connector-test-kit/src/test/scala/writes/unchecked_writes/UncheckedCreateSpec.scala

```diff
@@ -0,0 +1,385 @@
+package writes.unchecked_writes
+
+import org.scalatest.{FlatSpec, Matchers}
+import util._
+
+// Important: This test covers ALL top level create inputs, like create & upsert,
```

Comment:
Shouldn't we call it `UncheckedOperationsSpec` then?

Reply:
No, we shouldn't.